fabfile.py

```diff
@@ -13,13 +13,41 @@
 CURRENT_DIR = os.path.dirname(__file__)
 
 
+TIERS = {
+    'production': {
+        'hosts': ['root@104.131.92.59'],
```

Comment:
I, too, feel a little weird about having the username there, but I don't _think_ it's exposing us to much, if any, harm?